SignalServiceKit/src/Util/OWSAnalytics.m

```diff
@@ -399,7 +399,7 @@ - (void)logEvent:(NSString *)eventName
 
 - (BOOL)isSeverityAsync:(OWSAnalyticsSeverity)severity
```

Comment:
nit: this method name is not intuitive to me, maybe: `shouldReportAsync:` or invert it's usage to `shouldReportSync:`

Reply:
Donee.